### PR TITLE
add imaginary chart

### PR DESCRIPTION
**Pull request by FrancoisPoinsot**
add helm chart to deploy imaginary

### CODE REVIEW

File: charts/imaginary/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+appVersion: "1.0"
```
Comment (bueti) [inline charts/imaginary/Chart.yaml:2]: this should reflect the version of imaginary that is used

File: charts/imaginary/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Imaginary
+
+Imaginary repo: https://github.com/h2non/imaginary
+This chart is a fork of this PR: https://github.com/helm/charts/pull/6818
+
+## Config
+
+[values.yaml](./values.yaml)
```
Comment (bueti) [inline charts/imaginary/README.md:8]: you can auto-generate this README with [Helm-Docs](https://github.com/norwoodj/helm-docs). Check out https://github.com/ricardo-ch/helm-charts/blob/main/charts/promlens/README.md.gotmpl

File: charts/imaginary/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```
Comment (bueti) [inline charts/imaginary/Chart.yaml:1]: this should be apiVersion: v2 (helm 3)

File: charts/imaginary/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+home: https://github.com/h2non/imaginary
+sources:
+  - https://github.com/ricardo-ch/helm-charts/tree/main/charts/imaginary
+  - https://github.com/h2non/imaginary
+maintainers:
+  - name: ricardo-ch
+    email: sre@ricardo.ch
+keywords:
+  - imaginary
+  - h2non
```
Comment (bueti) [inline charts/imaginary/Chart.yaml:15]: add artifact-hub annotation:
```
annotations:
  artifacthub.io/changes: |
    - Initial release (blablabla)
  artifacthub.io/images: |
    - name: imaginary
      image: h2non/imaginary:v1.2
```

File: charts/imaginary/values.yaml [hunk 5]
```diff
@@ -0,0 +1,135 @@
+# A node selector label.
```
Comment (bueti) [inline charts/imaginary/values.yaml:1]: check https://github.com/norwoodj/helm-docs to see how to structure your comments to automatically generate parameter documentation

File: charts/imaginary/templates/pdb.yaml [hunk 6]
```diff
@@ -0,0 +1,14 @@
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  labels:
+    app: {{ template "imaginary.fullname" . }}
+    maintainer: sre
+    release: production
```
Comment (bueti) [inline charts/imaginary/templates/pdb.yaml:7]: we should make this configurable for a public chart

File: charts/imaginary/templates/pdb.yaml [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+  name: {{ template "imaginary.fullname" . }}-pdb
+  namespace: {{ .Release.Namespace }}
+spec:
+  minAvailable: 50%
```
Comment (bueti) [inline charts/imaginary/templates/pdb.yaml:11]: we should make this configurable for a public chart

**Review by bueti — APPROVED** [review]

File: charts/imaginary/templates/pdb.yaml (hunk 6 above)
Comment (FrancoisPoinsot) [reply charts/imaginary/templates/pdb.yaml:7]: actually I did
I forgot to remove it here?

File: charts/imaginary/templates/pdb.yaml (hunk 7 above)
Comment (FrancoisPoinsot) [reply charts/imaginary/templates/pdb.yaml:11]: ok

**Pull request merged**